### ISSUE TITLE
iron-fit-behavior accessibility issues 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-fit-behavior#9]
Audit results: 
[Warning] Text elements should have a reasonable contrast ratio (2)
[Warning] The web page should have a title that describes topic or purpose (1)
Not applicable tests (13)
Passing tests (2)

Contrast: Can you simply check to see if the "show private api"gray text is meeting contrast guidelines? It seems a bit light. 

I am not able to activate show private API / hide private API / collapse with just keyboard. If these are meant to be interactive, they must be keyboard navigable.

**Comment by morethanreal** [comment]
This seems to be the doc page? Please see https://github.com/PolymerElements/iron-component-page/issues/15
